namespace py system.backup.structs
namespace js system.backup.structs

struct TBackupFile {
	1: string file_name,
	2: string last_modified
}